src/components/Button/Button.tsx

```diff
@@ -102,23 +115,27 @@ export const Button: FC<ButtonProps> = ({
 }: ButtonProps) => {
   // aria-label atrribute usage
   // developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute
+  // 'disabled' removes button for screen readers, so for a11y it's best to visually make them disbaled and use aria-diabled instead
```

Comment:
Can we do further investigation in the next Sprint before removing the disabled prop please, because removing the disabled prop also prevents the disabled functionality.

Reply:
Sure, I based this off of a demo from an a11y meetup i went to recently but can explain my reasoning and try to find examples

Reply:
Basically it's because adding a disabled prop provides a different experience for sighted and non sighted users. For sighted users, when a button is disabled, it's greyed out and not possible to click on it. However this can be done with code and without the attribute. If the disabled attribute is present, the button can't receive focus and tab order, which is not the same experience if it's part of a form. So I used aria-disabled to tell the screen reader it's not possible to use yet, (https://www.w3.org/WAI/PF/aria/states_and_properties#aria-disabled) and made the button not clickable and styled as disabled visually

Reply:
and `aria-disabled` will be ignored if `disabled` is present

Reply:
But if this maybe still is not right I can remove what I did for now

Reply:
Okay cool, thanks for describing the change. I tested in Chrome and with my Mac screen reader assistant on, and the screen reader says _' dimmed'_  which will give non-sighted users the same experience as sighted users. Could you include the a11y link in the comment as well for future reference please, thanks.